.github/workflows/test.yml

```diff
@@ -30,12 +30,43 @@ jobs:
         uses: actions/checkout@v2
       - name: Zip Test Bin
         run: |
+          set -x
           zip ./testdata/test_bin.zip ./testdata/test_bin
+          zip ./testdata/test_additional_one.zip ./testdata/additional-zips/test_additional_one
+          zip ./testdata/test_additional_two.zip ./testdata/additional-zips/test_additional_two
+          zip ./testdata/test_additional_with_eula_one.zip ./testdata/additional-zips/test_additional_one ./testdata/additional-zips/NOT-EULA.txt
+          zip ./testdata/test_additional_with_eula_two.zip ./testdata/additional-zips/test_additional_two ./testdata/additional-zips/NOT-EULA.txt
+          zip ./testdata/actions-docker-build.zip ./testdata/actions-docker-build
       - name: Upload a test artifact.
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         with:
-          path: testdata/test_bin.zip
           name: test_bin.zip
+          path: testdata/test_bin.zip
```

Comment:
Nit: to make it a bit cleaner, we might be able to do this all in 1-2 steps via the `upload entire directory` option: https://github.com/actions/upload-artifact#upload-an-entire-directory